spec/site_prism/page_objects/page/organisations/ucas_contacts.rb

```diff
@@ -2,17 +2,26 @@ module PageObjects
   module Page
     module Organisations
       class UcasContacts < PageObjects::Base
+        class ContactDetail < SitePrism::Section
+          element :heading, ".govuk-summary-list__value"
+          element :details, ".govuk-summary-list__value .govuk-body"
+          element :change_link, ".govuk-summary-list__actions .govuk-link"
+        end
+
         set_url "/organisations/{provider_code}/ucas-contacts"
 
         element :flash, ".govuk-success-summary"
-        element :utt_contact, '[data-qa="provider__utt_contact"]'
-        element :web_link_contact, '[data-qa="provider__web_link_contact"]'
-        element :finance_contact, '[data-qa="provider__finance_contact"]'
-        element :fraud_contact, '[data-qa="provider__fraud_contact"]'
-        element :admin_contact, '[data-qa="provider__admin_contact"]'
-        element :gt12_contact, '[data-qa="provider__gt12_contact"]'
-        element :application_alert_contact, '[data-qa="provider__application_alert_contact"]'
-        element :send_application_alerts, '[data-qa="provider__send_application_alerts"]'
+
+        sections :contacts, ContactDetail, ".ucas-contact-list__row"
+        section :admin_contact, ContactDetail, "[data-qa='provider__admin_contact']"
+        section :utt_contact, ContactDetail, "[data-qa='provider__utt_contact']"
+        section :web_link_contact, ContactDetail, "[data-qa='provider__web_link_contact']"
+        section :finance_contact, ContactDetail, "[data-qa='provider__finance_contact']"
+        section :fraud_contact, ContactDetail, "[data-qa='provider__fraud_contact']"
+        section :gt12_contact, ContactDetail, "[data-qa='provider__gt12_contact']"
+
+        element :application_alert_contact, "[data-qa='provider__application_alert_contact']"
+        element :send_application_alerts, "[data-qa='provider__send_application_alerts']"
```

Comment:
Don't each of the summary list rows have a custom class name that we could hang off instead of using data-qa?

Reply:
They didn't but I've added one and removed the data-qa